src/main/kotlin/kotlinx/kover/tasks/KoverXmlTask.kt

```diff
@@ -0,0 +1,51 @@
+package kotlinx.kover.tasks
+
+import kotlinx.kover.engines.commons.*
+import org.gradle.api.file.*
+import org.gradle.api.tasks.*
+
+// TODO make internal in 0.7 version
```

Comment:
Why can't it be internal right now?

Reply:
If the class becomes internal, then the deprecated fields will become unavailable and the hint will not be shown, only `unresolved reference`.